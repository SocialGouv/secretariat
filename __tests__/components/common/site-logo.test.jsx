import { render } from "@testing-library/react"

import SiteLogo from "../../../src/components/common/site-logo"

import { it, expect } from "vitest"

it("renders regular site logo", () => {
  const { container } = render(<SiteLogo />)
  expect(container).toMatchSnapshot()
})

it("renders big site logo", () => {
  const { container } = render(<SiteLogo big={true} />)
  expect(container).toMatchSnapshot()
})
