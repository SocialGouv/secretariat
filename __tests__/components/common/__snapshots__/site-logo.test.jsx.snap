// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders big site logo 1`] = `
<div>
  <div
    class="logo big"
  >
    République
    <br />
    Française
  </div>
</div>
`;

exports[`renders regular site logo 1`] = `
<div>
  <div
    class="logo"
  >
    République
    <br />
    Française
  </div>
</div>
`;
